src/Controller/ExportController.php

```diff
@@ -42,32 +43,106 @@ public function beforeFilter(Event $event) : void
         }
     }
 
+    /**
+     * Export all data (per object type) in csv format
+     *
+     * @return void
+     */
+    public function exportAll() : void
+    {
+        // check request (allowed methods and required parameters)
+        $data = $this->checkRequest([
+            'allowedMethods' => ['post'],
+            'requiredParameters' => ['objectType'],
+        ]);
+
+        // load csv data for all objects by object type
+        $rows = $this->csvRows($data['objectType']);
+
+        // save data to csv and output it to browser
+        $this->csv($rows, $data['objectType']);
+    }
+
     /**
      * Export data in csv format
      *
      * @return void
      */
     public function export() : void
     {
-        if (empty($this->request)) {
-            return;
-        }
-        $this->request->allowMethod(['post']);
-        $ids = $this->request->getData('ids');
-        if (empty($ids)) {
-            return;
+        // check request (allowed methods and required parameters)
```

Comment:
why not have single `export` method and routing rule? `csvRows` method altready works this way

 * if no `ids` are set all objects - until max limit - are exported
 * if `ids` are set only selected elements